testnets/terratestnet/assetlist.json

```diff
@@ -0,0 +1,162 @@
+{
+  "$schema": "../assetlist.schema.json",
```

Comment:
since this is in /testnets/, the referenced schema requires an additional `../` (this applies to assetlist, chain, and versions .json files)

Reply:
No, like this:
`"$schema": "../../assetlist.schema.json",`